src/pages/home/report/ReportActionCompose/AttachmentPickerWithMenuItems.js

```diff
@@ -257,6 +261,7 @@ function AttachmentPickerWithMenuItems({
                             onClose={onPopoverMenuClose}
                             onItemSelected={(item, index) => {
                                 setMenuVisibility(false);
+                                onItemSelected();
```

Comment:
Can we move this to be the last function call?

Reply:
We need to overwrite `isKeyboardVisibleWhenShowingModalRef.current` value in `triggerAttachmentPicker` function when selecting `Add attachment`.
I think this is the ideal position this call should be placed.
Is there any specific reason for moving?